core/test/matrix/CMakeLists.txt

```diff
@@ -1,11 +1,12 @@
 ginkgo_create_test(coo)
 ginkgo_create_test(csr)
 ginkgo_create_test(dense)
+ginkgo_create_test(diagonal)
 ginkgo_create_test(ell)
 ginkgo_create_test(hybrid)
 ginkgo_create_test(identity)
 ginkgo_create_test(permutation)
 ginkgo_create_test(sellp)
 ginkgo_create_test(sparsity_csr)
 ginkgo_create_test(csr_builder)
-ginkgo_create_test(coo_builder)
\ No newline at end of file
+ginkgo_create_test(coo_builder)
```

Comment:
I believe this file should be sorted. Never saw before that `coo_builder` and `csr_builder` were at the very end